crates/burn-core/src/module/param/tensor.rs

```diff
@@ -147,6 +150,22 @@ impl<const D: usize, B: Backend> Module<B> for Param<Tensor<B, D>> {
     }
 }
 
+impl<const D: usize, B: Backend> ModuleDisplayDefault for Param<Tensor<B, D>> {
```

Comment:
Could you have a single implementation generic over the kind?

Reply:
I get this error:

```
the trait bound `burn_tensor::Tensor<B, D, K>: param::base::Parameter` is not satisfied
the trait `param::base::Parameter` is not implemented for `burn_tensor::Tensor<B, D, K>`
```

So to take advantage of individual implementation, I have added tensor kind. This what it looks like now:

```
Whisper {
  encoder: Encoder {
    conv1: Conv1d {stride: 1, kernel_size: 3, dilation: 1, groups: 1, padding: Explicit(1)}
    conv2: Conv1d {stride: 2, kernel_size: 3, dilation: 1, groups: 1, padding: Explicit(1)}
    positional_embedding: ParamTensor {rank: 2, shape: [1500, 384], kind: float, id: usor44svfc}
 ```